.changeset/eighty-carpets-smile.md

```diff
@@ -0,0 +1,5 @@
+---
+'@voucherify/sdk': patch
+---
+
+Added customers tests
```

Comment:
Why 2 identical changesets? 😅 
The tests aren't shipped in the dist and this PR doesn't affect the code output in the library, so maybe leave the changeset empty?:
```suggestion
---
---

Added customers tests
```